src/results.js

```diff
@@ -31,20 +32,66 @@ class Err {
 	get isErr() { return true }
 	get ok() { return null }
 	get err() { return this._error }
+
 	toString() {
 		if (this._error)
 			return "Error: " + JSON.stringify(this._error)
-		else
-			return "Error"
+		return "Error"
 	}
+
 	toJSON() {
 		let error = (this._error === undefined ? '' : this._error)
 		error = error instanceof Error ? error.toString() : error
 		return { 'Error': error }
 	}
+
+	static _notFound ({ message = 'Not Found', payload, stackTrace } = {}){ 
+		return Err._buildCustomErr('NOT_FOUND', message, payload, stackTrace, 'NotFound', 'notFound')
+	}
+	
+	static _alreadyExists ({ message = 'Entity already exists', payload, stackTrace } = {}) { 
```

Comment:
I think `stackTrace` is for exceptions, not errors. Instead, we should have `exception` arg for the exception that triggered the error (valid for all new errors here) and the stack trace should be inside the exception thrown.

Reply:
`message = 'Already exists',` so it can be used in different context other than for a entity